<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DAChartLegendItemSettingWidget</class>
 <widget class="QWidget" name="DAChartLegendItemSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Chart Legend Item Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Position</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="DA::DAAligmentPositionEditWidget" name="widgetPosition" native="true"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Horizontal Offset In Canvas</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxHorizontalOffset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Vertical Offset In Canvas</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBoxVerticalOffset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Margin around legend items</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Spacing between the legend items</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBoxSpacing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Margin around each item</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QSpinBox" name="spinBoxItemMargin">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Spacing inside of each item</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSpinBox" name="spinBoxItemSpacing">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Shape</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Max columns</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
        <property name="maximum">
         <double>999.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="DA::DAFontEditPannelWidget" name="widgetFont" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Radius for the border</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Font</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="DA::DAPenEditWidget" name="widgetBorderPen" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxMaxColumns">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Border pen</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Background</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="DA::DABrushEditWidget" name="widgetBKBrush" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DA::DAPenEditWidget</class>
   <extends>QWidget</extends>
   <header>DAPenEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAFontEditPannelWidget</class>
   <extends>QWidget</extends>
   <header>DAFontEditPannelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DABrushEditWidget</class>
   <extends>QWidget</extends>
   <header>DABrushEditWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DA::DAAligmentPositionEditWidget</class>
   <extends>QWidget</extends>
   <header>DAAligmentPositionEditWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
